src/screens/WebView/TermsAndCondition.js

```diff
@@ -30,7 +31,7 @@ const styles = StyleSheet.create({
   container: {
     paddingHorizontal: 20,
     flex: 1,
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hardcoded color values with centralized color constants improves consistency and maintainability of the codebase. Good job on this change.
```diff
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->